autoload/compe/documentation.vim

```diff
@@ -2,36 +2,59 @@ let s:MarkupContent = vital#compe#import('VS.LSP.MarkupContent')
 let s:FloatingWindow = vital#compe#import('VS.Vim.Window.FloatingWindow')
 
 let s:window = s:FloatingWindow.new()
+let s:border = s:FloatingWindow.new()
 
 "
 " compe#documentation#show
 "
 function! compe#documentation#open(document) abort
+  call compe#documentation#close()
   if !compe#_is_selected_manually()
     return
   endif
 
   let l:document = split(s:MarkupContent.normalize(a:document), "\n", v:true)
-
-  let l:pos = s:get_screenpos(pum_getpos(), l:document)
-  if empty(l:pos)
-    return s:window.close()
+  let l:layout = s:get_screenpos(pum_getpos(), l:document)
+  if empty(l:layout)
+    return
   endif
 
+  let l:maxwidth = float2nr(&columns * 0.4)
+  let l:maxheight = float2nr(&lines * 0.4)
+
   call s:window.open({
-  \   'row': l:pos[0],
-  \   'col': l:pos[1],
-  \   'maxwidth': float2nr(&columns * 0.4),
-  \   'maxheight': float2nr(&lines * 0.4),
+  \   'row': l:layout.row + 1,
+  \   'col': l:layout.col + 1,
+  \   'maxwidth': l:maxwidth - 2,
+  \   'maxheight': l:maxheight - 2,
   \   'filetype': 'markdown',
   \   'contents': l:document,
+  \   'winhl': 'NormalFloat:Normal',
+  \ })
+
```

Comment:
Add something like `chars = vim.g.compe.border.chars or [defaults]` to enable use customization.

Reply:
Good.

```
let g:compe = {}
let g:compe.border = v:false
let g:compe.border = {}
let g:compe.border.chars = {}
let g:compe.border.chars.top_left = ''
let g:compe.border.chars.top_right = ''
let g:compe.border.chars.bottom_left = ''
let g:compe.border.chars.bottom_right = ''
let g:compe.border.chars.vertical = ''
let g:compe.border.chars.horizontal = ''
```

I don't plan to add colors etc for now.

Reply:
I'd say make it an array of chars [bottom_corner_l, bottom_corner_r, top_corner_r, top_corner_l, top/bottom, left/right]

so the user can do [char, char, ... ] in specific order.